widget/release_notes.md

```diff
@@ -1,2 +1,8 @@
 # What's New
-Minor bug fix:
+Minor bug fixes.
+ - Widget now remembers a change in 'Picklist' value.
```

Comment:
Minor bug fixes:

Reply:
Remembers?

Reply:
'Updates'?

Reply:
Check with namarata, what is a better word here.